frontend/src/pages/modelRegistrySettings/CreateMRSecureDBSection.tsx

```diff
@@ -102,6 +89,7 @@ export const CreateMRSecureDBSection: React.FC<CreateMRSecureDBSectionProps> = (
       nameSpace: '',
       key: '',
       configMap: '',
```

Comment:
Total nit, we can keep it as-is if you like - the `configMap` key in SecureDBInfo is really the name of a configmap or a secret. I wonder if now that we have `resourceType` we should rename this to `resourceName`.